lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -160,6 +160,31 @@ function getById(id, callback) {
     });
 }
 
+function find(service, subservice, callback) {
```

Comment:
JavaDoc missing?

(The "?" is because I have doubts... on one hand no other function in this file includes JavaDoc, but maybe is a good idea to to at least for newly added functions)